### PR TITLE
Add "notes" field for request issues

### PR DESCRIPTION
**Pull request by leikkisa**
connects #6986

### CODE REVIEW

**Comment by leikkisa** [comment]
screen shot 2018-10-03 at 6 27 08 pm

File: local/vacols/bgs_setup.csv [hunk 1]
```diff
@@ -18,7 +18,7 @@ vbms_id,bgs_key
 767574947S,
 209179363S,
 626343664S,
-849386058S,
+849386058S,has_many_ratings
```
Comment (pkarman) [inline local/vacols/bgs_setup.csv]: I have re-introduced these locally as well but I'm not sure we want them back in the repo? I don't know why they were removed in the first place.

File: client/app/intake/components/AddIssuesModal.jsx [hunk 2]
```diff
@@ -24,8 +26,14 @@ class AddIssuesModal extends React.Component {
...
 
+  notesOnChange = (value) => {
+    this.setState({
+      notes: value
+    });
+  }
+
   onAddIssue = () => {
-    this.props.addIssue(this.state.referenceId, this.props.ratings, true);
+    this.props.addIssue(this.state.referenceId, this.props.ratings, true, this.state.notes);
```
Comment (pkarman) [inline client/app/intake/components/AddIssuesModal.jsx]: this many arguments to a method seems to beg for named pairs instead.
```javascript
this.props.addIssue({refId: this.state.referenceId, ratings: this.props.ratings, isRated: true, notes: this.state.notes})
```

File: local/vacols/bgs_setup.csv (hunk 1 above)
Comment (leikkisa) [reply local/vacols/bgs_setup.csv]: Will clean this up for now. I learned that this file should be generated by cases.csv, but re-generating it requires using another RAILS_ENV and haven't gotten around to trying it yet :-P.

**Comment by leikkisa** [comment]
Yeah I asked about displaying the notes in the UI afterwards and didn't get a response, but my intuition is that we should put them there.

**Comment by leikkisa** [comment]
screen shot 2018-10-04 at 9 17 22 am

**Pull request merged**